Src/SDK/Common/Temporal.Common.Payloads/public/PayloadContainers.Enumerable.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
+using Newtonsoft.Json;
 using Temporal.Util;
 
 namespace Temporal.Common.Payloads
 {
     public static partial class PayloadContainers
     {
+        [JsonConverter(typeof(PayloadContainersEnumerableJsonConverter))]
```

Comment:
It didn't work in the unit test I created, so that is why I added the converter. If the assumptions in the test are wrong, then this is unnecessary. The tests, I believe, were for the Newtonsoft json converter.

Reply:
Yes, Newtonsoft. Let's chat on Monday and figure it out.

Reply:
Found the underlying issue and resolved the issue by adding proper treatment for JObject. JsonConverter no longer needed and removed.